core/store/tx_manager.go

```diff
@@ -313,9 +327,10 @@ func (txm *EthTxManager) sendInitialTx(
 }
 
 var (
-	nonceTooLowRegex = regexp.MustCompile("(nonce .*too low|same hash was already imported)")
+	nonceTooLowRegex = regexp.MustCompile("(nonce .*too low|same hash was already imported|replacement transaction underpriced)")
```

Comment:
@spooktheducks I believe you had some comments here, we definitely still want this right?

Reply:
Does it do the right thing in the case that the node operator manually retried the TX with a higher gas price? I feel like maybe not...